redash/query_runner/cass.py

```diff
@@ -45,14 +68,43 @@ def configuration_schema(cls):
                     "default": 3,
                 },
                 "timeout": {"type": "number", "title": "Timeout", "default": 10},
+                "useSsl": {"type": "boolean", "title": "Use SSL", "default": False},
+                "sslCertificateFile": {
+                    "type": "string",
+                    "title": "SSL Certificate File"
+                },
+                "sslProtocol": {
+                    "type": "string",
```

Comment:
You can use an enum here. You can see an example in the MongoDB query runner. Although it uses an extended enum type, which allows for setting a title for each option. There is also a simpler type, but I don't have an example for it :|

Reply:
Yeah, I wanted to find if there were enum like option for the configuartion but didn't know exactly which library or config parser you are using, and didn't dig deeper. Will look at the Mongo Example and set it up.

Reply:
I had trouble using the extended Enum, Throws errors when saving the form. I tried the MongoDb connection form, and it has the same issue. So I am using plain enum and it seems to work.

Reply:
Sorry about this, the issue with extendedEnum is fixed in https://github.com/getredash/redash/pull/4774.